nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Mb employee edit

# Description

Edit works, including computerzzzz.

Closes #16

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout mb-employee-edit`
4. edit employee. see if she twerks.

# Checklist:
